doc/governance.rst

```diff
@@ -53,24 +53,21 @@ the CLT itself.
 Current CLT members are:
 
  * Abhishek Lekshmanan <abhishek@suse.com>
- * Alfredo Deza <adeza@redhat.com>
  * Casey Bodley <cbodley@redhat.com>
  * Gregory Farnum <gfarnum@redhat.com>
- * Haomai Wang <haomai@xsky.com>
  * Jason Dillaman <dillaman@redhat.com>
  * Josh Durgin <jdurgin@redhat.com>
  * João Eduardo Luis <joao@suse.de>
  * Ken Dreyer <kdreyer@redhat.com>
- * Lenz Grimmer <lgrimmer@suse.com>
  * Matt Benjamin <mbenjami@redhat.com>
- * Myoungwon Oh <omwmw@sk.com>
  * Neha Ojha <nojha@redhat.com>
  * Patrick Donnelly <pdonnell@redhat.com>
  * Sage Weil <sage@redhat.com>
  * Sebastian Wagner <swagner@suse.com>
- * Xie Xingguo <xie.xingguo@zte.com.cn>
  * Yehuda Sadeh <yehuda@redhat.com>
- * Zack Cerza <zcerza@redhat.com>
+ * Xie Xingguo <xie.xingguo@zte.com.cn>
```

Comment:
thought the names were ordered alphabetically. seems we are using a new ordering algorithm here?

Reply:
@tchaikov Nah, they're supposed to be alphabetical. I was going to alphabetize this list this morning after my meeting with Neha and Josh, but then I grabbed a hot cast iron skillet with my bare hand and, as a result, have been *hors d'combat* for the past few hours. Stand by and I'll get this fixed.